Resources/config/jackalope.xml

```diff
@@ -40,16 +36,12 @@
 
         <service id="doctrine_phpcr.jackalope.repository.factory.prismic"
                  class="Jackalope\Repository"
-                 factory-service="doctrine_phpcr.jackalope.repository.factory.service.prismic"
-                 factory-method="getRepository"
                 >
             <argument type="collection"/>
         </service>
 
         <service id="doctrine_phpcr.jackalope.session"
                  class="Jackalope\Session"
-                 factory-service="doctrine_phpcr.jackalope.repository"
-                 factory-method="login"
                  abstract="true"
         >
             <argument /><!-- phpcr credentials service (0) -->
```

Comment:
this line is really strange and causing the error when running `vendor/symfony-cmf/testing/bin/travis/../console doctrine:phpcr:init:dbal --drop --force`. Unless, I'm [missing something](https://github.com/jackalope/jackalope/blob/1.1/src/Jackalope/Session.php#L126), `Jackalope\Session` wants the PHPCR credentials as the 4th argument, not as the 1st.